scheduler/src/cook/mesos/api.clj

```diff
@@ -435,29 +486,27 @@
                     start (:instance/start-time instance)
                     end (:instance/end-time instance)
                     reason (reason/instance-entity->reason-entity db instance)
-                    base {:task_id (:instance/task-id instance)
-                          :hostname hostname
-                          :ports (:instance/ports instance)
-                          :backfilled (:instance/backfilled? instance false)
-                          :preempted (:instance/preempted? instance false)
-                          :slave_id (:instance/slave-id instance)
-                          :executor_id (:instance/executor-id instance)
-                          :status (name (:instance/status instance))}
-                    base (if url-path
-                           (assoc base :output_url url-path)
-                           base)
-                    base (if start
-                           (assoc base :start_time (.getTime start))
-                           base)
-                    base (if end
-                           (assoc base :end_time (.getTime end))
-                           base)
-                    base (if reason
-                           (assoc base
-                                  :reason_code (:reason/code reason)
-                                  :reason_string (:reason/string reason))
-                           base)]
-                base))
+                    code (:instance/code instance)
```

Comment:
Can we rename this (and other uses of it) from code to exit-code so there is no confusion what is being referred to?